packages/vuetify/src/components/VItemGroup/_variables.scss

```diff
@@ -1,3 +1,3 @@
 @import '../../styles/styles.sass';
```

Comment:
Update sass structure, add index.scss and adjust imports (same as btn and other components)